apps/cookbook/src/app/showcase/item-showcase/item-showcase.component.html

```diff
@@ -158,6 +158,12 @@ <h3 #multiLine>Multiline</h3>
     #multiLineExample
   ></cookbook-item-example-simple-multiline>
 </cookbook-example-viewer>
+<h3>Multiline - subtitle</h3>
```

Comment:
Consider adding a note in somewhere in this "Simple" section where we showcase `subtitle` and `detail` concepts, that recommends using the new classes instead of attributes.

Reply:
👍🏻 
https://github.com/kirbydesign/designsystem/pull/3734/commits/befb194c8e05c58b6c19c8b74d4de7eea1d65157